### PR TITLE
Clean up style notes from flake8

### PR DESCRIPTION
**Pull request by bbayles**
Re: issue #4, this PR gets the codebase passing `flake8` checks.

To do this I ran `black`, which does reasonably good re-formatting. Then I fixed up the remaining issues.

(Since `black` only runs on Py3 I actually ran `modernize`, then `black`, then `flake8`, and then dropped the `six` stuff added by the first tool)

### CODE REVIEW

File: pure_protobuf/protobuf.py [hunk 1]
```diff
@@ -409,78 +509,89 @@ def __init__(self, message_type):
...
         '''
-        (self.__dict__ if name in self.__dict__ else self).__setitem__(name, value)
+        mapping = self.__dict__ if name in self.__dict__ else self
+        mapping.__setitem__(name, value)
+
         return value
-    
-    def __delattr__(self, name):
+
+    def __delattr__(self, name, value):
```
Comment (bbayles) [inline pure_protobuf/protobuf.py]: This seems like it was real bug; I'll add a test for it in a different PR.

**Comment by eigenein** [comment]
Couple of things regarding `__delattr__` and `__delitem__`, otherwise looks good for me.

**Comment by eigenein** [comment]
Could you also add a `CHANGELOG.md` item?

**Comment by bbayles** [comment]
Re: `__delattr__`, good spot.

**Comment by bbayles** [comment]
I had confused myself about the `__delattr__` change. I've now un-confused myself, actually fixed the bug, and added a test.

**Pull request merged**